src/transformers/__init__.py

```diff
@@ -103,6 +103,7 @@
         "is_clearml_available",
         "is_comet_available",
         "is_neptune_available",
+        "is_optimum_available",
```

Comment:
This is laready in `import_utils`.

Reply:
Fixed in [9c5e0d9](https://github.com/huggingface/transformers/pull/21259/commits/9c5e0d9308b5c4882805058023729fb6dd63cbe9)